Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -1191,8 +1191,7 @@ extension System {
                     walletManagerEvent = WalletManagerEvent.syncEnded(reason: reason)
 
                 case CRYPTO_WALLET_MANAGER_EVENT_BLOCK_HEIGHT_UPDATED:
-                    manager.network.height = event.u.blockHeight.value
-                    // No event?
+                    walletManagerEvent = WalletManagerEvent.blockUpdated(height: event.u.blockHeight.value)
```

Comment:
The block height is consistently set at the Crypto C level?

Reply:
```
case BITCOIN_WALLET_MANAGER_BLOCK_HEIGHT_UPDATED: {
            cwmEvent = (BRCryptoWalletManagerEvent) {
                CRYPTO_WALLET_MANAGER_EVENT_BLOCK_HEIGHT_UPDATED,
                { .blockHeight = { event.u.blockHeightUpdated.value }}
            };
            BRCryptoNetwork network = cryptoWalletManagerGetNetwork (cwm);
            cryptoNetworkSetHeight (network, event.u.blockHeightUpdated.value);
            cryptoNetworkGive (network);
            break;
        }

case EWM_EVENT_BLOCK_HEIGHT_UPDATED: {
            cwmEvent = (BRCryptoWalletManagerEvent) {
                CRYPTO_WALLET_MANAGER_EVENT_BLOCK_HEIGHT_UPDATED,
                { .blockHeight = { event.u.blockHeight.value }}
            };
            BRCryptoNetwork network = cryptoWalletManagerGetNetwork (cwm);
            cryptoNetworkSetHeight (network, event.u.blockHeight.value);
            cryptoNetworkGive (network);
            break;
        }
```